redis.conf

```diff
@@ -540,7 +540,7 @@ dir ./
 #
 # masterauth <master-password>
 #
-# However this is not enough if you are using Redis ACLs (for Redis version
```

Comment:
Valkey first version is 7.2, here, we could write it comes from the Redis 6 or greater or Valkey 7.2